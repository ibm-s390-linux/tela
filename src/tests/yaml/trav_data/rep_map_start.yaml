replace:
  child1:
map2:
  child2:
