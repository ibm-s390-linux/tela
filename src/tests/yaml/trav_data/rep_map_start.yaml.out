Before (root=non-null):
==================================
  replace:
    child1:
  map2:
    child2:
==================================

Callback:
==================================
replace                                 : replace:
*** Replacing node
map2                                    : map2:
map2/child2                             :   child2:
==================================

After (root=non-null):
==================================
  replacement
  map2:
    child2:
==================================
